packages/ui/react/src/components/Forms/Label/Label.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { Tooltip } from '../../Primitives'
+import { LabelElement, ContentFlex, QuestionIcon } from './Label.styled'
+import { getColor } from 'src/colors'
+
+type LabelProps = {
+  /** Main label text/node */
+  value: React.ReactNode
+  /** @deprecated if helpValue is present help is considered true */
+  help?: boolean
+  /** Help text shown in tooltip */
+  helpValue?: string
+  /** ARIA role */
+  role?: string
+  /** Adds "(Optional)" text */
+  isOptional?: boolean
+}
+
+/**
+ * Label component with optional help tooltip
+ *
+ * @example
+ * ```jsx
+ * // Basic label
+ * <Label value="Username" />
+ *
+ * // Label with help text and optional marker
+ * <Label
+ *   value="Email Address"
+ *   helpValue="We'll send a verification code"
```

Comment:
Can you update the example?

```suggestion
 *   value="Description"
 *   helpValue="A description of the item"
```